packages/smart-wallet/src/proposals/upgrade-walletFactory-proposal.js

```diff
@@ -0,0 +1,128 @@
+// @ts-check
+import { E } from '@endo/far';
+import { makeMarshal } from '@endo/marshal';
+import { allValues } from '@agoric/internal';
+
+console.warn('upgrade-walletFactory-proposal.js module evaluating');
+
+const { Fail } = assert;
+
+// vstorage paths under published.*
+const WALLET_STORAGE_PATH_SEGMENT = 'wallet';
+const BOARD_AUX = 'boardAux';
+
+const marshalData = makeMarshal(_val => Fail`data only`);
+
+/**
+ * @param { BootstrapPowers } powers
+ *
+ * @param {object} config
+ * @param {{ walletFactoryRef: VatSourceRef & { bundleID: string } }} config.options
+ */
+export const upgradeWalletFactory = async (
+  {
+    consume: {
+      contractKits,
+      governedContractKits,
+      chainStorage,
+      walletBridgeManager: walletBridgeManagerP,
+    },
+    instance: {
+      consume: { walletFactory: wfInstanceP, provisionPool: ppInstanceP },
+    },
+  },
+  config,
+) => {
+  console.log('upgradeWalletFactory: config', config);
+  const { walletFactoryRef } = config.options;
+
+  // console.log('upgradeWalletFactory: awaiting instances etc.');
```

Comment:
We should try not to have commented-out code.

Please uncomment or remove. I suggest uncomment as its appropriate for proposals to be verbose.